README.md

```diff
@@ -137,6 +138,7 @@ Your contributions are always welcome!
 - [Quran-Corpus](http://corpus.quran.com/) :  Quranic Arabic Corpus, an annotated linguistic resource which shows the Arabic grammar, syntax and morphology for each word in the Holy Quran. The corpus provides three levels of analysis: morphological annotation, a syntactic treebank and a semantic ontology.
 - [Hadith-islamware](https://github.com/ceefour/hadith-islamware) : Hadith database from Islam Ware https://www.islamware.com/app/downloads
 - [PDF Quran](https://github.com/abodehq/QuranPDF) : PDF version for Quran in different languages
+- [Open-Hadith-Data-Project](https://github.com/mhashim6/Open-Hadith-Data-Project) : An open Hadith Library that contains full databases of 9 different books.
```

Comment:
I'm terribly sorry for the late response,
Yes, I meant this project, and as for the other project, it's not gonna be available at the moment.
thanks.